backend/app/assessment/events.py

```
@@ -0,0 +1,51 @@
+    async def subscribe(self) -> AsyncIterator[str]:
+        queue: asyncio.Queue[dict] = asyncio.Queue()
+        self._subscribers.add(queue)
+        logger.info(
+            "[subscribe] New SSE subscriber | total=%d", len(self._subscribers)
+        )
+        try:
+            yield "event: ready\ndata: {}\n\n"
+            while True:
+                try:
+                    payload = await asyncio.wait_for(queue.get(), timeout=15)
+                except asyncio.TimeoutError:
```

Ruff (0.15.12)

[warning] Replace aliased errors with `TimeoutError`

Replace `asyncio.TimeoutError` with builtin `TimeoutError`

(UP041)